### ISSUE TITLE
ValidClassNameSniff should check also simple classes

### ISSUE THREAD

**Issue by gmponos** [djoos/Symfony-coding-standard#140]
I believe that `ValidClassNameSniff` should also check simple classes (T_CLASS) and display an error if they have `abstract` `interface`, `exception` and `trait`

If I have some time I will work on it if someone else does not do this before me.

**Comment by wickedOne** [comment]
hi there @gmponos,

could you clarify your issue?

``T_INTERFACE``, ``T_TRAIT`` and ``T_ABSTRACT`` should catch the interface, trait and abstract classes.

if you mean that instances of 
```php
class FooInterface {
    // ...
}
```
should raise a warning: not sure whether i agree as that's more a coding issue than a code style violation...

**Comment by djoos** [comment]
Closing this one off for now - happy to revisit the discussion once @gmponos provides further clarification...

**Comment by gmponos** [comment]
> if you mean that instances of
> ```
> class FooInterface {    // ... }
> ```
> 
> should raise a warning: not sure whether i agree as that's more a coding issue than a code style violation...

That's what I meant..
